lib/iterator.js

```diff
@@ -199,6 +199,19 @@ class Iterator {
     return result + suffix;
   }
 
+  // Split iterator into chunks of size `from` or every time function
+  // `from` returns falsy value.
+  //   from - <Function> | <number> split function or chunk size
+  //     val <any> current iterator value
+  chunks(from) {
```

Comment:
IMHO, this should use a verb for a name.

Reply:
So how about `chunk` or `chunkBy`?

Reply:
Either one is ok with me.